app/controllers/v1/resources_controller.rb

```diff
@@ -49,6 +49,7 @@ def write_purl
       end
 
       write_public_cocina
+      write_public_purl_metadata(purl)
```

Comment:
This should go into the `ReleasedController#update` method.

Reply:
I think we'll want it in both places, to ensure we always have meta information even for newly released/never released objects, right?